packages/universal-components/src/RangeDatePicker/DayItemArrow/DayItemArrow.android.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { StyleSheet } from '../../PlatformStyleSheet';
+import { ExtendedTouchable } from '../../ExtendedTouchable';
+import { type Props } from './DayItemArrowTypes';
+
+export default class DayItemArrow extends React.Component<Props> {
+  renderArrow = () => (
+    <View style={styles.touchableContainer}>
+      <View
+        style={[
+          styles.arrow,
+          this.props.direction === 'left'
+            ? styles.arrowLeft
+            : styles.arrowRight,
+        ]}
+      />
+    </View>
+  );
+
+  render() {
+    return (
+      <View
+        style={[
+          styles.container,
+          this.props.direction === 'left'
+            ? styles.containerArrowLeft
+            : styles.containerArrowRight,
+          this.props.style,
+        ]}
+      >
+        {this.props.onPress ? (
+          <ExtendedTouchable overlap={20} onPress={this.props.onPress}>
```

Comment:
I'm just thinking if we even need the arrow to be touchable on mobile. Probably just drag-select functionality like in current kiwi mobile app can be enough.

Reply:
Yes, that's the plan, but I wanted to solve first possibility of picking range dates and then implement gestures.

Reply:
Ok, I was just thinking that if it causes too much trouble on Android we can probably skip this functionality on that platform for now. But since you have it already solved, we can keep it as it is.